robocop/checkers/comments.py

```diff
@@ -9,6 +11,15 @@
 from robocop.checkers import RawFileChecker, VisitorChecker
 from robocop.rules import Rule, RuleSeverity, RuleParam
 from robocop.utils import ROBOT_VERSION
+from robocop.exceptions import ConfigGeneralError
+
+def regex(value):
+    converted = rf'{value}'
+    try:
+        re.compile(converted)
```

Comment:
You're not returning compiled pattern, but converted string instead. I guess it should be:
```
try:
    return re.compile(converted)
    except re.error as regex_err:
        raise ValueError(f'Regex error: {regex_err}')
```

Reply:
fixed.